data/json/professions.json

```diff
@@ -5138,5 +5138,145 @@
                 "panties"
             ]
         }
+    },
+    {
+        "type": "profession",
+        "ident": "escape_con",
+        "name": "Escaped Prisioner",
+        "description": "When the cataclysm started you took your chance, now you roam looking for a place to stay. Hopefully those days workin in prison pay off..",
+        "points": -2,
```

Comment:
Needs a very good justification for the cost. -2 is what broken borg gets.

Reply:
Can I just, like make it 0?